### PR TITLE
001 view job roles

### PR DESCRIPTION
**Pull request by autumncraig**
Updated - focused on US001 and only has Job Role column in Job Role table. UI has homepage plus Job Role page which should link to API and display data from Job Role table. 
Screenshot 2023-12-12 at 09 28 27

### CODE REVIEW

File: .github/workflows/Simple.yml
Comment (jkelso662) [inline .github/workflows/Simple.yml:1]: What are these emply workflows all about? Always best not to commit empty files unless for good reason.

File: eslintrc.js
Comment (jkelso662) [inline eslintrc.js:1]: Same thing here

**Comment by lullabelle** [comment]
Hi!
There are some extensions in gitignore that you are probably not using on this project so you can remove those (e.g, grunt, gatsby etc!)

File: pullRequestTemplate
Comment (jkelso662) [inline pullRequestTemplate:1]: If this works, ignore me - but I assumed this file had to be named something specific with a specific file extension?

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Few comments to checkout here. Also theres no screenshot on the description so no idea what this UI looks like. Should there be any unit tests? Food for thought.

**Pull request closed without merging**